.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,62 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-production-image:
+    name: Build and test production docker image
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Create docker production image
+      run: docker build . --tag=openslides-backend
+
+    - name: Run docker production image
+      run: docker run --detach --publish 9002:9002 --publish 9003:9003 openslides-backend
+
+    - name: Sleep for some seconds
+      run: sleep 2
```

Comment:
Can this be removed? it is very arbitrary. If you want to wait for your services to be online, use another tool.

Reply:
I will open an issue so that we can insert the usage of another tool later. I do not know a smart solution at the moment.

Reply:
The question is: Why did you add it? What goes wrong when you leave it out?

Reply:
The previous command finishs before the server is really ready. The next command comes to quickly.

Reply:
You can use `wait.sh <host> <port>` for this case.